python/server/auto_generated/qgsserversettings.sip.in

```diff
@@ -193,6 +193,17 @@ Returns ``True`` if the reading flag trust layer metadata is activated.
 The default value is ``False``, this value can be changed by setting the environment
 variable QGIS_SERVER_TRUST_LAYER_METADATA.
 
+.. versionadded:: 3.16
+%End
+
+    bool getPrintDisabled() const;
+%Docstring
+Returns ``True`` if the reading flag don't load layouts is activated and
```

Comment:
I am having a hard time reading this. Is it my English or is this not English?

Reply:
I have change the description:
```
Returns TRUE if WMS GetPrint request is disavbled and the project's reading flag DONT_LOAD_LAYOUTS is activated.
```